Assignment 2.Rmd

```diff
@@ -81,19 +86,82 @@ plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender)
 
 plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender, edge.width=EDGE$count)
```

Comment:
Did you delete the cleaning steps in Part 1? When you run the code, it seems stuck.